packages/shared/src/modules/friends/libs/enums/friend-api-path.enum.ts

```diff
@@ -0,0 +1,10 @@
+const FriendApiPath = {
+	FOLLOW: "/follow",
+	FOLLOWERS: "/followers",
+	FOLLOWINGS: "/followings",
+	POTENTIAL_FOLLOWING: "/potential-following",
```

Comment:
```suggestion
	POTENTIAL_FOLLOWINGS: "/potential-followings",
```
I think we will receive more than one?